atmo/forms/mixins.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, *args, **kwargs):
         self.required_filefields = []
 
         field_order = []
-        for name, field in self.fields.items():
+        fields = self.fields.copy()
```

Comment:
Wait, I think my later use of `list` on the fields negated that need, so we can stop copying it.